src/components/Dashboard/Events/EventsBody.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { CardBody } from 'patternfly-react';
+
+const EventsBody = ({ children, ...props }) => (
+  <CardBody {...props} className="kubevirt-events__body">
```

Comment:
@rawagner I tried using div and it's not working. It expects class cardBody's class to be in same div.

If it's coming like this[1] then its working. but when using div the `card-pf-body` is a parent at that point it's not working. 
1. `<div id="events-body" class="card-pf-body kubevirt-events__body" style="position: relative;">`

2. 
```
<div class="card-pf-body>
    <div id="events-body" class="card-pf-body kubevirt-events__body" style="position: relative;"> 
    </div>
</div>
```

Reply:
@cloudbehl I tried it and found the issue
1. div in EventsBody
```
<div id="events-body" className="kubevirt-events__body">
    {children}
  </div>
```

2. `.kubevirt-events__body` css style needs to have fixed height `height: 400px` instead of currently used `max-height`. Also it would be good to change `margin` to `margin: -15px -15px;`

Reply:
@rawagner Ack, will make the change and I will update the PR.

Reply:
@rawagner I provided the min-height and the max-height. Instead of just passing the height.
If there is only one Event in the box then we will show to the our min-height. Does it sounds good?

Also due to the margin change the processing/loaded icon is going out of context.

Reply:
Sounds good. I will fix the loading state in followup PR